app/assets/stylesheets/beta.css.scss

```diff
@@ -0,0 +1,42 @@
+@import "compass";
+@import "colors";
+
+.beta-signup {
+  margin-top: 30%;
+  width: 100%;
+
+  .fi-laptop {
+    font-size: 1.8em;
+    margin-right: 3px;
+    vertical-align: middle;
+  }
+}
+
+#beta_signin {
+  .button {
+    border: $cmPurple;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Name of variable `cmPurple` should be written in all lowercase letters with hyphens instead of underscores